src/ShapeCatalog.js

```diff
@@ -38,7 +38,11 @@ class ShapeHero extends Component {
 }
 
 class ShapeDemos extends Component {
-  ripples = [];
+  constructor(props) {
+    super(props);
+    this.ripples = [];
```

Comment:
On next PR, we don't need to wrap this in the `constructor`. These could just live on the class like so:

```js
class ShapeDemos extends Component {
  ripples = [];
  initRipple = surfaceEl => surfaceEl && this.ripples.push(new MDCRipple(surfaceEl));
}
```

Reply:
When it's created this way, is ripples still effectively an instance variable? I was worried it was being created on the class.

At any rate, this matches exactly what we were already doing in ButtonCatalog. There's probably a few things we could clean up in general at some point in the future.

Reply:
They are instance variables, here is a codepen showing that https://codepen.io/moog16/pen/ZoJVrR.

But yes, there are a few places that we created those variables in constructors, so we'll need to clean up eventually.